ohmec_data_ma.js

```diff
@@ -32,51 +32,1926 @@ dataNA = {
+		[[[-80.80627,41.79984],[-80.05388,40.49003],[-79.90053,39.51868],[-80.18618,38.49425],[-81.12114,37.89521],[-81.98793,36.66601],[-82.07582,35.85101],[-81.10903,34.84741],[-80.31113,32.52273],[-79.49516,32.69808],[-78.76977,33.81642],[-78.04467,33.83468],[-77.84692,34.28974],[-77.14409,34.69057],[-76.55053,34.70624],[-76.30883,34.92272],[-76.57251,34.97675],[-76.79223,35.46141],[-76.11108,35.35396],[-75.71557,35.80073],[-76.59448,36.03207],[-76.13335,36.17644],[-75.73754,36.10311],[-75.9353,36.75722],[-76.24292,36.8276],[-76.37475,37.14353],[-76.22094,37.80616],[-76.96801,38.30789],[-76.39672,38.06611],[-76.55053,39.284],[-75.84741,39.52169],[-76.19897,39.13077],[-76.177,38.48011],[-75.6936,37.92758],[-75.84741,37.37088],[-75.05639,38.42849],[-75.53979,39.50474],[-74.9685,39.11372],[-74.94653,38.8233],[-74.68315,39.21819],[-74.22143,39.48779],[-73.97973,40.29698],[-74.13384,40.58349],[-73.71606,40.56459],[-72.19995,40.96399],[-73.71606,40.9474],[-72.72759,41.29719],[-71.25512,41.39398],[-70.85961,41.62434],[-70.42016,41.72281],[-70.59594,42.19664],[-71.07934,42.26172],[-70.77172,42.60229],[-70.83764,42.941],[-70.02465,43.72413],[-69.12377,43.99347],[-68.8601,44.40304],[-68.55249,44.23011],[-68.2229,44.52849],[-68.04711,44.34022],[-67.80571,44.57754],[-67.36626,44.57754],[-66.90454,44.71616],[-67.12426,45.10519],[-67.45385,45.24459],[-67.43188,45.56855],[-67.78344,45.67612],[-67.78344,47.0857],[-68.15727,47.34141],[-68.94799,47.16046],[-69.12377,47.4287],[-69.25591,47.46039],[-69.98071,46.68022],[-70.04663,46.43848],[-70.28833,46.19567],[-70.24438,45.81412],[-70.74975,45.44535],[-70.7937,45.21364],[-70.96948,45.30644],[-71.38696,45.22912],[-71.38696,45.02759],[-74.88061,45.02759],[-75.67192,44.56189],[-76.33081,44.13557],[-76.15532,44.15342],[-76.19897,43.88271],[-76.24292,43.46953],[-77.01196,43.26187],[-78.46215,43.38974],[-79.05542,43.21385],[-78.87963,42.84442],[-78.9895,42.55375],[-80.80627,41.79984]]]]}},
+
+//500B.C. - 500A.D.
+{ "type":"Feature",
+      "id":"ALBH",
+      "style":{
+	"fillColor": "#83AA9B", "strokeColor": "#83AA9B"}, 
+      "properties":{
+        "entity1type":"culture",
+        "entity1name":"Archaic Level Bison Hunters",
+        "fidelity":"XXXX",
```

Comment:
These should all be fidelity 1, for the cultures and subcultures.

Reply:
Should I change the fidelity of the archeological sites as well?